qt/widgets/common/src/ManageUserDirectories.cpp

```diff
@@ -82,15 +100,21 @@ void ManageUserDirectories::loadProperties() {
   m_uiForm.leDefaultSave->setText(saveDir);
 }
 void ManageUserDirectories::saveProperties() {
-  QString newSearchArchive;
-  QString newDataDirs;
-  QString newUserDirs;
-  QString newSaveDir;
-
-  if (m_uiForm.ckSearchArchive->isChecked())
-    newSearchArchive = "On";
-  else
-    newSearchArchive = "Off";
+  QString newSearchArchive = m_uiForm.cbSearchArchive->currentText().toLower();
+  if (newSearchArchive == "all") {
+    // do nothing
+  } else if (newSearchArchive == "none") {
+    newSearchArchive = "off";
+  } else if (newSearchArchive.startsWith("default facility only")) {
+    newSearchArchive = "on";
+  } else {
+    // the only way "custom" gets set is by using the value in ConfigService
+    // already, so just copy it
+    QString::fromStdString(Mantid::Kernel::ConfigService::Instance().getString(
+                               "datasearch.searcharchive"))
```

Comment:
You are prepending `"custom - "` to the custom option above which is getting saved, after opening ManageUserDirectories a few time my properties file looks like
```
datasearch.searcharchive=custom - custom - custom - custom - custom - sns
```
😀